targets/targets.json

```diff
@@ -1030,6 +1030,22 @@
         "release_versions": ["2", "5"],
         "device_name": "STM32F401RE"
     },
+    "STEVAL_3DP001V1": {
+        "inherits": ["FAMILY_STM32"],
+        "core": "Cortex-M4F",
+        "extra_labels_add": ["STM32F4", "STM32F401xE", "STM32F401VE"],
+        "config": {
+            "clock_source": {
+                "help": "Mask value : USE_PLL_HSE_EXTC | USE_PLL_HSE_XTAL (need HW patch) | USE_PLL_HSI",
+                "value": "USE_PLL_HSE_EXTC|USE_PLL_HSI",
```

Comment:
I don't know the board, but is HSE from ST Link connected ?
Or are you using HSI ?

Reply:
My knowledge here is lacking. So not sure what you mean (I just copied this piece of code from the NUCLEO_F303RE).
There is a 25Mhz xtal connected to PH0/PH1. That seems to be the only clock connected.

Reply:
OK
Could you check (RCC->PLLCFGR & RCC_PLLCFGR_PLLSRC) value ?
I expect 0, ie. HSI, which means that HSE configuration has failed.